.github/ISSUE_TEMPLATE.md

```diff
@@ -1,13 +1,18 @@
+##Actual Behavior
```

Comment:
I think we should share a single heading for both Steps to reproduce: and Actual Behavior:. The more headings we ask here the less likely the user will fill the things we are asking him/her.